### PR TITLE
Feature/header component

### PR DESCRIPTION
**Pull request by MTCawthray**
# Thank you for pushing your code!
## What (if any) features are you implementing?
- This branch creates the components that will make up our header- Header and NavBar.
- This adds basic styling to the header just to get away from the default styling that React provides. These changes are NOT set in stone at this point!
- This adds 3 buttons to the nav bar- ```my profile``` ```friends``` and ```log out```

## What (if anything) did you refactor?
- Removed the hard coded header JSX from the return of the App component and replaced it with the header component

## Were there any issues that arose?
- NA

## Is there anything that you need from your teammates?
- Again, these CSS stylings were only to change it up from default. Fonts/colors/layouts/etc are open game for later in the project.

## Any other comments, questions, or concerns?
- There is currently no functionality associated with the nav buttons. Once we implement React-Router we can make these into NavLinks and add other functionality! 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor

## How Has This Been Tested?
- [x] open index.html
- [ ] console.log()
- [x] dev tools
- [ ] Cypress

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/Header/Header.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+h1 {
+  margin: 1vh 0 0 0;
+  color: #ff3b3f;
+  font-weight: 800;
+  text-shadow: 1px 0px rgb(119 119 119 / 74%);
+}
+
+.App-header {
```
Comment (hayleyw7) [inline src/components/Header/Header.css:8]: Thanks for making this look great!

File: src/components/NavBar/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './NavBar.css';
+
+const NavBar = () => {
```
Comment (hayleyw7) [inline src/components/NavBar/NavBar.js:4]: Appreciate you getting this set up!

**Review by hayleyw7 — APPROVED** [review]

File: src/components/Header/Header.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
+h1 {
+  margin: 1vh 0 0 0;
+  color: #ff3b3f;
+  font-weight: 800;
+  text-shadow: 1px 0px rgb(119 119 119 / 74%);
```
Comment (nataliemcintyre2021) [inline src/components/Header/Header.css:5]: Love the shadowing!

File: src/components/NavBar/NavBar.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        <button className="nav-btn">friends</button>
+      </li>  
+      <li>
+        <button className="nav-btn">log out</button>
+      </li>
+    </ul>
+  )
+}
+
+export default NavBar;
```
Comment (nataliemcintyre2021) [inline src/components/NavBar/NavBar.js:21]: Love the styling - makes for great user experience.

**Pull request merged**